src/js/contentScript.js

```diff
@@ -15,8 +15,7 @@
 
 			if (top.nodeType === Node.ELEMENT_NODE && regex.test(top.innerHTML)) {
 				top.innerHTML = top.innerHTML.replace(regex, replaceMethod.bind(top));
-			}
-			else if (top.nodeType === Node.TEXT_NODE && regex.test(top.nodeValue)) {
+			} else if (top.nodeType === Node.TEXT_NODE && regex.test(top.nodeValue)) {
```

Comment:
I think we should make an eslint config for this project.. so that we all are on the same page.